demisto_sdk/commands/common/hook_validations/layout.py

```diff
@@ -65,6 +66,45 @@ def is_to_version_higher_than_from_version(self) -> bool:
                     return False
         return True
 
+    @staticmethod
+    def get_invalid_incident_fields_from_tabs(layout_tabs, content_fields):
+        """
+        Get the invalid incident fields from layout tabs.
+
+        Args:
+            layout_tabs (list): a list of the layout tabs.
+            content_fields (list): a list of the content field items.
+
+        Returns:
+            list[str]: incident fields which do not exist in the content items.
+        """
+        non_existent_incident_fields = []
+
+        for tab in layout_tabs:
+            layout_sections = tab.get('sections', [])
+            for section in layout_sections:
+                items = section.get('items', [])
+                non_existent_incident_fields.extend(
+                    get_invalid_incident_fields_from_layout(layout_incident_fields=items, content_fields=content_fields)
+                )
+
+        return non_existent_incident_fields
+
+    @staticmethod
+    def get_fields_from_id_set(id_set_file: dict) -> list:
```

Comment:
do we know type of dict here?

Reply:
do you mean the types off keys/values?

Reply:
yeah, can we put Dict[str,str] here? I wasnt sure what the types are

Reply:
its ```Dict[str, List]``` ill change